Source/ZoomNet/Models/CallHandlingSettings/CallHandlingSubsettings.cs

```diff
@@ -0,0 +1,322 @@
+// Ignore Spelling: Voicemail
+
+using System.Text.Json.Serialization;
+
+namespace ZoomNet.Models.CallHandlingSettings
+{
+	/// <summary>
+	/// The model of call handling subsettings.
+	/// </summary>
+	public class CallHandlingSubsettings : CallHandlingSubsettingsBase
+	{
+		/// <summary>
+		/// Gets or sets a value indicating whether to allow the callers to check voicemails.
+		/// This field is only available in the following scenarios:
+		/// - The <see cref="CallNotAnswerAction"/> is set to 1 (Forward to a voicemail).
+		/// - The <see cref="BusyOnAnotherCallAction"/> is set to 1 (Forward to a voicemail).(Only applicable to the User)
```

Comment:
```suggestion
		/// - The <see cref="BusyOnAnotherCallAction"/> is set to 1 (Forward to a voicemail).(Only applicable to the User).
```

Satisfy SA1629: Documentation text should end with a period